src/Acuminator/Acuminator.Tests/Verification/VerificationHelper.cs

```diff
@@ -28,9 +30,23 @@ public static class VerificationHelper
 		private static readonly MetadataReference PXDataReference = MetadataReference.CreateFromFile(typeof(PXGraph).Assembly.Location);
 		private static readonly MetadataReference FluentBqlReference = MetadataReference.CreateFromFile(typeof(FbqlCommand).Assembly.Location);
 		private static readonly MetadataReference PXCommonReference = MetadataReference.CreateFromFile(typeof(PX.Common.PXContext).Assembly.Location);
+		private static readonly MetadataReference PXCommonStdReference = MetadataReference.CreateFromFile(typeof(PX.Common.PXInternalUseOnlyAttribute).Assembly.Location);
 		private static readonly MetadataReference PXObjectsReference =
 			MetadataReference.CreateFromFile(typeof(PX.Objects.GL.PeriodIDAttribute).Assembly.Location);
 
+		private static readonly MetadataReference[] _metadataReferences =
```

Comment:
Since you moved all the references to an array, do we still need them as separate `static readonly` properties? I think the whole array can consist of `MetadataReference.CreateFromFile(...)` and similar calls to increase readability - right now a developer need to follow the links 2 times (`AddMetadataReferences` -> `_metadataReferences` -> a particular private field)  to get the idea of what references are actually added to the project.

Reply:
The logic is a bit more complex - there is a `BuildAssemblyFromSources` method which creates a separate `MetadataReference` array with almost all of this references present (excluding `FluentBqlReference`). However, I think it's possible to change the code and use the same metadata references everywhere. So I'll fix it

Reply:
I have tried to make an array of `MetadataReference.CreateFromFile` and actually I don't think that it will be more helpful for the developer. When I look at the 
```csharp
MetadataReference.CreateFromFile(typeof(PX.Common.PXInternalUseOnlyAttribute).Assembly.Location)
```
I don't know what assembly I'll get. For the majority of the symbols I don't know which assemblies contain which types and imho other devs also don't remember this. If I try to guess the assembly for `PX.Common.PXInternalUseOnlyAttribute` I would say that it is located in 'PX.Common' due to namespace name. But it isn't this way now. I need to make the same "Go to" to the `PX.Common.PXInternalUseOnlyAttribute` symbol's metadata to see its assembly. Or I have to write assembly name in the comment to keep the info about the assembly in the same location. Neither is better than existing solution in my opinion when we already have a meaningful names for the assembly fields

Reply:
I also dislike getting an assembly reference from a specific type. It seems more logical to use relative paths since it is more readable.

Reply:
But getting the reference from the type's assembly location is more stable - you can't get out of sync with file location. If you transfer the assembly files to some other place you have to remember that you must update the strings with paths to assemblies here. Moreover you can do this only for the Acumatica assemblies. It shouldn't be done with corlib assembly or System.Core or assemblies from nuget packages. So it won't be very consistent when about a half of assemblies is created with a string and a half with type's assembly location.